modules/library/referencing/src/main/java/org/geotools/referencing/operation/transform/GeocentricTranslation.java

```diff
@@ -154,7 +154,7 @@ public ParameterValueGroup getParameterValues() {
      * Creates an inverse transform using the specified matrix.
      */
     @Override
-    MathTransform createInverse(final Matrix matrix) {
+    GeocentricTranslation createInverse(final Matrix matrix) {
```

Comment:
I am a little uncomfortable changing the return type in a public API. 

Is GeocentricTranslation a kind of MathTransform?